lib/game_collection.rb

```diff
@@ -13,4 +13,15 @@ def create_games(file_path)
 
     csv.map { |row| Game.new(row) }
   end
+
+  def percentage_home_wins
+    home_wins = @games.find_all do |game|
+      game.home_goals > game.away_goals
+    end
+    (home_wins.length.to_f / @games.length).round(2)
+  end
+
+  
+
+
```

Comment:
Is this where the end was messing things up

Reply:
Yes. Fixed!